diff --git a/browser/themes/shared/jar.inc.mn b/browser/themes/shared/jar.inc.mn
index 14ef7ff731145a42b28cbb9afe192bf56f756ed5..9bfb7cdfaef3688c2064320f85b9196daf8a8215 100644
--- a/browser/themes/shared/jar.inc.mn
+++ b/browser/themes/shared/jar.inc.mn
@@ -15,7 +15,7 @@
   skin/classic/browser/addon-notification.css                  (../shared/addon-notification.css)
   skin/classic/browser/autocomplete.css                        (../shared/autocomplete.css)
   skin/classic/browser/blockedSite.css                         (../shared/blockedSite.css)
-  skin/classic/browser/browser-shared.css                      (../shared/browser-shared.css)
+* skin/classic/browser/browser-shared.css                      (../shared/browser-shared.css)
   skin/classic/browser/ctrlTab.css                             (../shared/ctrlTab.css)
   skin/classic/browser/light-dark-overrides.css                (../shared/light-dark-overrides.css)
   skin/classic/browser/menupanel.css                           (../shared/menupanel.css)
